.github/workflows/ci-gh1.yml

```diff
@@ -0,0 +1,60 @@
+name: Build legate.core on GH
+
+concurrency:
+  group: ci-gpu-on-${{ github.event_name }}-from-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches-ignore:
+      - gh-pages  # deployment target branch (this workflow should not exist on that branch anyway)
+  # NOTE: this workflow will not work for pull_requests since GH pull requests do not have access to secrets.
+
+jobs:
+  build:
+    permissions: read-all
+    # Ref: https://docs.rapids.ai/resources/github-actions/#cpu-labels for `linux-amd64-cpu4`
+    runs-on: ${{ github.repository == 'nv-legate/legate.core' && 'linux-amd64-cpu4' || 'ubuntu-latest' }}
+    container:
+      options: -u root
+      image: rapidsai/devcontainers:23.06-cpp-cuda11.8-mambaforge-ubuntu22.04
+      env:
+        PYTHONDONTWRITEBYTECODE: 1
```

Comment:
```suggestion
        JOBS: 1
        CMAKE_BUILD_PARALLEL_LEVEL: 1
        PYTHONDONTWRITEBYTECODE: 1
```

Reply:
Do we really want to hard code the number of parallel processes to 1? I think we are using the value returned by `nproc --ignore 1` to set this value. So this works out to 1 for GH runners anyway. Also it will work out to 3 for the 4 cpu runner.